VERSION

```diff
@@ -1 +1 @@
-0.1
\ No newline at end of file
+0.1.1
```

Comment:
Since it's a patch version bump, I think [`treeder/bump`](https://github.com/treeder/dockers/tree/master/bump#bump) will do it automatically as part of docker release, no?